dockstore-integration-testing/src/test/resources/dockstore.yml

```diff
@@ -81,3 +81,32 @@ logging:
       timeZone: UTC
       target: stdout
       logFormat: # TODO
+
+uiConfig:
+  # Must end with a slash
+  discourseUrl: https://discuss.dockstore.org/
+  dnaStackImportUrl: https://app.dnastack.com/#/app/workflow/import/dockstore
+  fireCloudImportUrl: https://portal.firecloud.org/#import/dockstore
+  dnaNexusImportUrl: https://platform.dnanexus.com/panx/tools/import-workflow
+  terraImportUrl: https://app.terra.bio/#import-tool/dockstore
+
+  gitHubAuthUrl: https://github.com/login/oauth/authorize
+  gitHubRedirectPath: /auth/github.com
+  gitHubScope: read:org,user:email
+
+  quayIoAuthUrl: https://quay.io/oauth/authorize
+  quayIoRedirectPath: /auth/quay.io
+  quayIoScope: repo:read,user:read
+
+  bitBucketAuthUrl: https://bitbucket.org/site/oauth2/authorize
+
+  gitlabAuthUrl: https://gitlab.com/oauth/authorize
+  gitlabRedirectPath: /auth/gitlab.com
+  gitlabScope: api
+
+  googleScope: profile email
```

Comment:
space separated?

Reply:
Yes, that's how it is in dockstore.model.ts